scripts/fast-reboot-dump.py

```diff
@@ -244,18 +245,17 @@ def generate_default_route_entries(filename):
 
     db.close(db.APPL_DB)
 
-    if len(default_routes_output) > 0:
-        with open(filename, 'w') as fp:
-            json.dump(default_routes_output, fp, indent=2, separators=(',', ': '))
-    else:
-        if os.path.isfile(filename):
-            os.unlink(filename)
+    with open(filename, 'w') as fp:
+        json.dump(default_routes_output, fp, indent=2, separators=(',', ': '))
 
 
 def main():
-    all_available_macs, map_mac_ip_per_vlan = generate_fdb_entries('/tmp/fdb.json')
-    arp_entries = generate_arp_entries('/tmp/arp.json', all_available_macs)
-    generate_default_route_entries('/tmp/default_routes.json')
+    root_dir = '/tmp'
+    if len(sys.argv) > 1 and os.path.isdir(sys.argv[1]):
+        root_dir = sys.argv[1]
```

Comment:
it is better to use argparser to add option, easy to use, give you options, and helps.

https://docs.python.org/2/howto/argparse.html

Reply:
I agree